src/main/java/studio/magemonkey/fabled/data/sql/SQLManager.java

```diff
@@ -0,0 +1,86 @@
+package studio.magemonkey.fabled.data.sql;
+
+import lombok.Getter;
+import studio.magemonkey.fabled.Fabled;
+import studio.magemonkey.fabled.data.sql.tables.FabledPlayersSQL;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class SQLManager {
+
+    @Getter
+    private static volatile Connection connection;
+
+    private static FabledPlayersSQL fabledPlayersSQL;
+
+    private static final String host = Fabled.getSettings().getSqlHost();
+    private static final int port = Integer.parseInt(Fabled.getSettings().getSqlPort());
+    private static final String database = Fabled.getSettings().getSqlDatabase();
+    private static final String user = Fabled.getSettings().getSqlUser();
+    private static final String password = Fabled.getSettings().getSqlPass();
+
+    public static void reconnect() {
+        Fabled.inst().getLogger().info("Initializing SQLManager with type: MySQL");
+        connection = getMySQLConnection(host, port, database, user, password);
+        if (connection == null) {
+            Fabled.inst().getLogger().severe("Failed to initialize the Connection.");
+        }
+    }
+
+    public static void reload() {
```

Comment:
I think this was misinterpreted a little. The reload method would technically be called upon `/mirage reload` and would pull the host/user/etc from the config file anew in case the database configuration was changed.

Reply:
Reworked in the last commit 👍